README.md

```diff
@@ -18,7 +18,7 @@ Gherkin 3 is currently implemented for the following platforms:
 * Go
 * Python
 
-See `CONTRIBUTING.md` if you want to contribute a parser for a new language.
+See [`CONTRIBUTING.md`](CONTRIBUTING.md) if you want to contribute a parser for a new language.
```

Comment:
Please don't make unrelated changes in the same PR. This PR is about improving string handling, yet you're also improving the docs.

You've also done similar (conflicting?) changes in #99.